Lesson_Materials/Lecture_12_Advanced_Data_Flow/index.html

```diff
@@ -189,60 +189,24 @@
 						* A `buffer` with no final data address is useful because the data can left on a device and not copied back to the host from the device.
 					</div>
 				</section>
-				<!--Slide 11-->
-				<section>
-					<div class="hbox" data-markdown>
-						#### Uninitialized buffers (buffer/accessor)
-					</div>
-					<div class="container">
-						<code class="code-100pc"><pre>
-auto buf = sycl::buffer{sycl::range{size}};
-						</code></pre>
-					</div>
-					<div class="container" data-markdown>
-						* As we've seen in the USM model all memory is allocated initialized, but `buffer`s can be constructed within initial data.
-						* Alternatively a `buffer` can be constructed as uninitialized.
-						* To do this simply construct a `buffer` without initial data.
-						* Uninitialized `buffer`s are useful for a couple of reasons because they can be allocated directly on a device and not copied to the device from the host.
-					</div>
-				</section>
 				<!--Slide 12-->
 				<section>
 					<div class="hbox" data-markdown>
 						#### Uninitialized buffers (buffer/accessor)
 					</div>
 					<div class="container">
 						<code class="code-100pc"><pre>
-auto buf = sycl::buffer{sycl::range{size}};
-						</code></pre>
-					</div>
-					<div class="container" data-markdown>
-						* As we've seen in the USM model all memory is allocated initialized, but `buffer`s can be constructed within initial data.
-						* Alternatively a `buffer` can be constructed as uninitialized.
-						* To do this simply construct a `buffer` without initial data.
-						* Uninitialized `buffer`s are useful for temporary data or data which is initialized within a kernel function.
-					</div>
-				</section>
-				<!--Slide 13-->
-				<section>
-					<div class="hbox" data-markdown>
-						#### Uninitialized buffers (buffer/accessor)
-					</div>
-					<div class="container">
-						<code class="code-100pc"><pre>
-auto buf = sycl::buffer{sycl::range{size}};
-
-buf.set_final_data()
+auto buf = sycl::buffer&lt;T&gt;{sycl::range{size}};
 						</code></pre>
 					</div>
 					<div class="container" data-markdown>
-						* As we've seen in the USM model all memory is allocated initialized, but `buffer`s can be constructed within initial data.
-						* Alternatively a `buffer` can be constructed as uninitialized.
-						* To do this simply construct a `buffer` without initial data.
+						* As we've seen in the USM model all memory is allocated initialized, but `buffer`s can be constructed without initial data.
+						* A `buffer` like this is called uninitialized.
+						* To do this simply construct a `buffer` without initial data. Just remember to explicitly specify the buffers data type as it can't be inferred from the initial data anymore.
```

Comment:
```suggestion
						* To do this simply construct a `buffer` without initial data. Just remember to explicitly specify the buffer's data type as it can't be inferred from the initial data any more.
```